timeless/views.py

```diff
@@ -178,6 +180,19 @@ def get_context_object_list_name(self):
         """
         return self.context_object_list_name
 
+    def sort_query(self, query):
+        ordering = request.args.get("ordering")
+        if ordering:
+            for field_name in ordering.split(","):
+                order_direction = desc if field_name.startswith("-") else asc
```

Comment:
@vladarefiev I actually think filtering from the user's perspective should not be so complex. I think we should only allow filtering by one field at a time, ascending or descending... after all what does a use understand from "Order ascending by id and descending by firstName"? 

Then we also make our life easier with the implementation :)

Reply:
@amihaiemil One more time didn't get your point perfectly. I implemented some logic on back-end, it doesn't matter for me how front-end will use it, how much fields will be allowed for using. I just want to provide a mechanism, I'm waiting for a some query in GET params, for example:
`/?order_by=restaurant, -created_at`

Pls clarify more details ;)

Reply:
@vladarefiev the implemented logic accounts for ordering by multiple fields, some ascending, some descending: a user usually orders a table by a single column (ascending or descending), right?

What would be the output of this:  ``/?order_by=restaurant,-created_at``? A table ordered alphabetically by the restaurant's name... and at the same time descending by the creation date?

Reply:
@amihaiemil As I can see you want to think about business logic, but here I am trying to provide only back-end for correct sorting by fields ;)